esmvaltool/preprocessor/_reformat.py

```diff
@@ -51,7 +57,8 @@ def fix_metadata(cube, short_name, project, model, cmor_table=None, mip=None):
     return cube
 
 
-def fix_data(cube, short_name, project, model, cmor_table=None, mip=None):
+def fix_data(cube, short_name, project, model, preproc_dir,
```

Comment:
The new code here seems unrelated to fixing the data in the cube, but it looks like it is trying to clean up after the function `fix_file`. This should probably be removed.

Reply:
Yes, to avoid having the fixed files forever in the folder. At which step do you suggest that we clean them?

Reply:
I think it's probably best to create a separate `cleanup` preprocessor function, which we run at the very end, instead of adding it to some unrelated preprocessor function.

Reply:
Ok, I will add it